plugin.rb

```diff
@@ -85,14 +105,20 @@ def user_voted
       Category.can_vote?(object.id)
     end
 
+    register_search_advanced_filter(/^min_vote_count:(\d+)$/) do |posts, match|
+      posts.where("(SELECT COUNT(*) FROM discourse_voting_votes WHERE discourse_voting_votes.topic_id = posts.topic_id) >= ?", match.to_i)
+    end
+
+    register_search_advanced_order(:votes) do |posts|
+      posts.reorder("COALESCE((SELECT dvvc.counter FROM discourse_voting_vote_counters dvvc WHERE dvvc.topic_id = subquery.topic_id), 0) DESC")
+    end
+
     class ::Category
       def self.reset_voting_cache
         @allowed_voting_cache["allowed"] =
           begin
             Set.new(
```

Comment:
We don't need this set anymore since there is a unique index on the `catgegory_id` column. That does raise an interesting question though, do we have duplicated records that might cause the add unique index to fail?